docs/components/modeler/feel/builtin-functions/feel-built-in-functions-boolean.md

```diff
@@ -47,11 +46,93 @@ is defined(1)
 // true
 
 is defined(null)
-// true
+// false
 
 is defined(x)
 // false - if no variable "x" exists
 
 is defined(x.y)
 // false - if no variable "x" exists or it doesn't have a property "y"
 ```
+
+:::caution Breaking change
```

Comment:
As this is a breaking change, should this be noted anywhere else? Like announcements, or update material?

Reply:
I mentioned this change in the update guide [here](https://github.com/camunda/camunda-docs/pull/2658/files#diff-0da863f7f864baad0f87fef1747fb29bcda9f68da564fcec08bd4e2544f1782cR82-R92). :+1: 

For me, the update guide seems to be the best place for it. It is mainly relevant to updating from a previous version.

For an announcement, the change is probably too small and too much detail.